### PR TITLE
Feature open ssh port on pubip

### PR DESCRIPTION
**Pull request by sk4zuzu**

### CODE REVIEW

File: tests/tests.sh [hunk 1]
```diff
@@ -180,6 +208,8 @@ function validate-azure-resources-presence() {
   echo "#	will test if there is expected amount of machines in resource group"
   local vms_count=$(az vm list --subscription "$4" --resource-group azbi-module-tests-rg -o yaml | yq r - --length)
   if [[ $vms_count -ne 2 ]]; then exit 1; fi
+  local nsg_nic_count=$(az network nsg show --subscription "$4" --resource-group azbi-module-tests-rg --name vm-nic-nsg-0 -o yaml | yq r - networkInterfaces --length)
+  if [[ $nsg_nic_count -ne 2 ]]; then exit 1; fi
```
Comment (mkyc) [inline tests/tests.sh:212]: can you add ssh connection testing after this line?

**Review by mkyc — CHANGES_REQUESTED** [review]

File: tests/tests.sh (hunk 1 above)
Comment (sk4zuzu) [reply tests/tests.sh:213]: Please check in latest commits. :+1:

File: resources/terraform/modules/vms/variables.tf [hunk 2]
```diff
@@ -37,7 +37,7 @@ variable "tf_key_path" {
 
 variable "vm_size" {
   type = string
-  default = "Standard_DS1_v2"
+  default = "Standard_DS2_v2"
```
Comment (mkyc) [inline resources/terraform/modules/vms/variables.tf:40]: we need to create task that this should be parametrised.

**Review by mkyc — APPROVED** [review]
LGTM

File: resources/terraform/modules/vms/variables.tf (hunk 2 above)
Comment (sk4zuzu) [reply resources/terraform/modules/vms/variables.tf:40]: :+1: I just added this so we don't repeat the same mistakes.

Comment (seriva) [reply resources/terraform/modules/vms/variables.tf:40]: Also these should;d be configurable per machine.

Comment (mkyc) [reply resources/terraform/modules/vms/variables.tf:40]: Yeah, issue #13 is created.

**Review by seriva — APPROVED** [review]

**Pull request merged**